Set _FORTIFY_SOURCE to 3 to avoid a whole slew of

  <command-line>: warning: "_FORTIFY_SOURCE" redefined

messages during the build.

--- a/build/moz.configure/toolchain.configure
+++ b/build/moz.configure/toolchain.configure
@@ -2921,7 +2921,7 @@ def security_hardening_cflags(
         # Also, undefine it before defining it just in case a distro adds it, see Bug 1418398
         if compiler_is_gccish and optimize and not asan:
             flags.append("-U_FORTIFY_SOURCE")
-            flags.append("-D_FORTIFY_SOURCE=2")
+            flags.append("-D_FORTIFY_SOURCE=3")
 
         # Lib c++ debug mode
         if cxx_assert:
